### ISSUE TITLE
Add a command to migrate a service published in testnet to the mainnet

### ISSUE THREAD

**Issue by vsbogd** [singnet/snet-cli#154]
To make it simpler publishing services to the mainnet I think it makes sense to add a command which migrates the service published in testnet to the mainnet.

Then the process of publishing services may be the following:
- publish your service to the testnet;
- test the service configuration in the testnet;
- propagate the service to the mainnet.

**Comment by astroseger** [comment]
It is already very simple to do. You simply need to use ``` snet service print_metadata``` and ```snet service publish```. 
Let's consider the following situation
* You have your service published in kovan ( testo/tests ).
* You have your kovan identity ```user-kovan``` linked to kovan network.
* You have your mainnet identity ```user-mainnet``` linked to mainnet network.

```
# switch to kovan
snet identity user-kovan

# get metadata 
snet service print_metadata testo tests > service_metadata.json

# switch to mainnet
snet identity

# publish your service in main net
snet service publish <mainnet-org-id> <mainnet-serv-id>
````

It is best we can do, because having single function would require working in two networks simultaneously which would be difficult. 

So I propose to close this issue here. We could reopen it on wiki to add this explanation in the docs.

**Comment by astroseger** [comment]
@vsbogd very good point.
You are right. There are two options:
- We add special function to update mpe_address
- We indeed update it automatically in publish.

**Comment by vsbogd** [comment]
I would propose updating it automatically during publishing and may be have an option to skip this update and set `mpe_address` manually.

**Comment by astroseger** [comment]
Automatic update makes sense, because mpe-address is really technical field, it should be set automatically. But I'm warring that it  breaks semantic of "publish" function a little bit. Publish supposed to be read only...  Or isn't it?

**Comment by vsbogd** [comment]
I think there is always should be some manual way of passing `mpe-address` in case it is needed, for instance if contract was upgraded but user still uses old contract version.

Regarding `publish` semantics I would say the following. We need to give user a simple way of promoting service from testnet to mainnet. If `publish` command semantics doesn't allow doing it then we need to add separate command to do it. If we don't want to provide separate command we need to change `publish` command semantics. (btw, in any case `publish` should warn the user if it tries to use incorrect MPE address when he publishes service)

**Comment by astroseger** [comment]
closed with #158
